.github/workflows/build.yml

```diff
@@ -98,19 +98,14 @@ jobs:
     strategy:
       matrix:
         compiler: [gcc, clang]
-        version: [9, 10, 11, 12, 13, 14]
+        version: [11, 12, 13, 14]
```

Comment:
Question: Do you intended to not run clang 9 and clang 10 anymore?

Reply:
We excluded them before already (see further down). But we don't do gcc 9 and 10 anymore. It think that's fine. I don't think running these old gcc on macos give us much information at this point.

Reply:
Sorry, I meant gcc 9, and gcc 10.